addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1493,6 +1493,10 @@ registry.WebsiteAnimate = publicWidget.Widget.extend({
         this.$scrollingElement[0].classList.remove('o_wanim_overflow_xy_hidden');
     },
 
+    getScrollingElement() {
+        return $().getScrollingElement();
+    },
```

Comment:
It seems nice like this but I am planning to remove the `getScrollingElement` entirely in a near future (we don't need it anymore since 18.0 since the scroll is not on the wrapwrap ever anymore).

Detail but could you name your new function `findScrollingElement` instead, that way I will be less annoyed while adapting :pray: